src/stan/variational/families/normal_fullrank.hpp

```diff
@@ -135,6 +135,37 @@ class normal_fullrank : public base_family {
    */
   const Eigen::MatrixXd& L_chol() const { return L_chol_; }
 
+  /**
+   * Return the approximation family's parameters as a single vector
+   */
+  Eigen::VectorXd return_approx_params() const {
```

Comment:
This part assumes that the only relevant parameters would be the lower triangular elements of the matrix. Am I correct?

Reply:
Looks like it. I wouldn't worry so much about fullrank till the diagonal one is working tho.